### PR TITLE
Remove bloom filter

### PR DESCRIPTION
**Pull request by PoignardAzur**
The filter was only used for RouteFocusChanged which is about to be removed. Since it was a performance feature, we can remove it early.

### CODE REVIEW

**Review by waywardmonkeys — CHANGES_REQUESTED** [review]
You will be able to remove the `fnv` dependency from both The masonry Cargo.toml but also the workspace one and Cargo.lock

**Comment by PoignardAzur** [comment]
Done.

File: xilem/examples/to_do_mvc.rs [hunk 1]
```diff
@@ -14,6 +14,7 @@ struct Task {
 
 struct TaskList {
     next_task: String,
+    next_task2: String,
```
Comment (waywardmonkeys) [inline xilem/examples/to_do_mvc.rs:17]: These changes to a xilem example seem unrelated and don’t belong in this PR.

**Review by waywardmonkeys — APPROVED** [review]
LGTM once the changes to the. Item example are removed.

File: xilem/examples/to_do_mvc.rs [hunk 2]
```diff
@@ -66,7 +66,7 @@ fn app_logic(task_list: &mut TaskList) -> impl WidgetView<TaskList> {
         })
         .collect::<Vec<_>>();
 
-    flex((first_line, tasks))
+    flex((first_line, input_box2, tasks))
```
Comment (waywardmonkeys) [inline xilem/examples/to_do_mvc.rs]: You still have this bit in there...

Comment (waywardmonkeys) [reply xilem/examples/to_do_mvc.rs]: You got it now.

**Pull request merged**